Tests/CommandsTests/RunToolTests.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import XCTest
+
+import TestSupport
+import Commands
+
+final class RunToolTests: XCTestCase {
+    private func execute(_ args: [String]) throws -> String {
+        return try SwiftPMProduct.SwiftRun.execute(args, printIfError: true)
+    }
+
+    func testUsage() throws {
+        XCTAssert(try execute(["--help"]).contains("USAGE: swift run [options] [executable <arguments>]"))
+    }
+
+    func testVersion() throws {
+        XCTAssert(try execute(["--version"]).contains("Swift Package Manager"))
+    }
```

Comment:
We should add a basic functional test. We can extend one of the executable related functional test.